client/js/verification-portal/components/header-modal.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react'
+import injectSheet from 'react-jss'
+import cx from 'classnames'
+import { connect } from 'react-redux'
+
+class HeaderModal extends Component {
+	constructor(props) {
+		super(props)
+		this.state = {
+			closed: false,
+			prevSubmit: null,
+		}
+		this.handleClick = this.handleClick.bind(this)
+	}
+
+	componentDidUpdate() {
+		const { submitStatus } = this.props
+		if (submitStatus === 'Pending' && this.state.closed) {
+			this.setState({
+				closed: false,
+			})
+		}
+
+		if (submitStatus !== 'Pending' && this.state.prevSubmit !== submitStatus) {
+			this.setState({
+				prevSubmit: submitStatus,
+			})
+		}
+	}
+
+	handleClick(e) {
+		e.preventDefault()
+		this.setState({
+			closed: true,
+		})
+	}
+
+	render() {
+		const { classes, submitStatus } = this.props
+		const modalClasses = cx(
+			classes.root,
+			submitStatus === 'Success' && classes.success,
+			submitStatus === 'Failed' && classes.error,
+			this.state.closed && classes.closed,
+		)
+
+		if (
+			(submitStatus === 'Success' && this.state.prevSubmit === null) ||
+			this.state.prevSubmit === 'Success'
+		) {
+			return (
+				<div className={modalClasses}>
+					<div>
+						<h2>Information Submitted Successfully</h2>
+						<p>
+							Thanks for helping us bring better healthcare to women's
+							fingertips!
+						</p>
+					</div>
+					<div className={classes.button} onClick={this.handleClick}>
+						&times;
+					</div>
+				</div>
+			)
+		} else {
+			return (
+				<div className={modalClasses}>
+					<div>
+						<h2>Whoops!</h2>
+						<p>An error occurred and the HAH staff has been notified.</p>
+					</div>
+					<div className={classes.button} onClick={this.handleClick}>
+						&times;
+					</div>
+				</div>
+			)
+		}
+	}
+}
+
+const styles = {
+	root: {
+		height: '0px',
+		color: '#FFFFFF',
+		display: 'flex',
+		'text-align': 'left',
+		background: '#ee6298',
+		'justify-content': 'center',
+		'align-items': 'center',
+		'font-size': '30px',
```

Comment:
It looks like the heading (1st line) text should be `37px` and the 2nd line should be `20px`